docker/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.9'
```

Comment:
a base docker-compose.yml should be nvidia-free ... another like docker-compose.cuda.yml can be nvidia based ... you can combine them running docker-compose